samples/caf_sensor_manager/app.overlay

```diff
@@ -9,6 +9,7 @@
 		compatible = "nordic,sensor-sim";
 		label = "SENSOR_SIM";
 		acc-signal = "wave";
+		status = "okay";
```

Comment:
Why is this relevant to this commit? The default status is `okay` if not defined, so this is redundant and does not seem related to the commit's purpose as I understand it.

Reply:
This was relevant because the way the sensor_sim is enabled now uses the default Kconfig setting that is delivered by devicetree check for this status. This however may not be relevant as I am having issues with this way of configuration in more complex architectures.

Reply:
I had changed the implementation for the configuration be only in Kconfig. The DTS now defines kind of "maximal" configuration. Please recheck.